src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -237,25 +238,43 @@ public VariantContext finalizeRawMQ(final VariantContext vc) {
         }
     }
 
-    protected void parseRawDataString(ReducibleAnnotationData<List<Integer>> myData) {
+    private void parseRawDataString(ReducibleAnnotationData<List<Long>> myData) {
         myData.putAttribute(Allele.NO_CALL, parseRawDataString(myData.getRawData()));
     }
 
     //TODO once the AS annotations have been added genotype gvcfs this can be removed for a more generic approach
-    private static List<Integer> parseRawDataString(String rawDataString) {
+    private static List<Long> parseRawDataString(String rawDataString) {
         try {
             final String[] parsed = rawDataString.split(",");
             if (parsed.length != NUM_LIST_ENTRIES) {
                 throw new UserException.BadInput("Raw value for annotation has " + parsed.length + " values, expected " + NUM_LIST_ENTRIES);
             }
-            final int squareSum = Integer.parseInt(parsed[SUM_OF_SQUARES_INDEX]);
-            final int totalDP = Integer.parseInt(parsed[TOTAL_DEPTH_INDEX]);
+            final long squareSum = Long.parseLong(parsed[SUM_OF_SQUARES_INDEX]);
+            final long totalDP = Long.parseLong(parsed[TOTAL_DEPTH_INDEX]);
             return Arrays.asList(squareSum,totalDP);
         } catch (final NumberFormatException e) {
             throw new UserException.BadInput("malformed " + GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY + " annotation: " + rawDataString);
         }
     }
 
+    /**
+     * Private getter function to replace VariantContext::getAttributeAsIntList in instances where there is a chance
+     * that ints will overlow beyond Integer.MAX_VALUE
+     * @return VariantContext attribute indexed by key, as list of long.
+     */
+    static private List<Long> getAttributeAsLongList(final VariantContext vc, final String key, final Long defaultValue) {
```

Comment:
I agree with your reticence to put this into htsjdk.  The type handling there is super awkward, but I don't think it's worth dealing with until it gets improved in 3.0.  However, I might suggest making this method public and moving it to a utility class.  `GATKProtectedVariantContextUtils` has a lot of similar methods